crosstargeting_override.props.sample

```diff
@@ -23,6 +23,7 @@
 		<!-- <TargetFrameworkOverride Condition="''!='hint: .NET 7.0 Android'">net7.0-android;$(TargetFrameworkOverride)</TargetFrameworkOverride> -->
 		<!-- <TargetFrameworkOverride Condition="''!='hint: .NET 7.0 macOS Catalyst'">net7.0-maccatalyst;$(TargetFrameworkOverride)</TargetFrameworkOverride> -->
 		<!-- <TargetFrameworkOverride Condition="''!='hint: .NET 7.0 macOS AppKit'">net7.0-macos;$(TargetFrameworkOverride)</TargetFrameworkOverride> -->
+		<TargetFrameworkOverride Condition="UseWinUI=='true'">net7.0-windows10.0.19041;$(TargetFrameworkOverride)</TargetFrameworkOverride>
```

Comment:
@kazo0 It should be commented on by default right and add a comment to add more info related to it no?
Can you give some feedback to guide @vatsashah45 on this one please

Reply:
It should just be a line like this: `<!-- <UseWinUI>True</UseWinUI> -->`

Same way we have a line for the NugetOverrideVersion here: https://github.com/unoplatform/Uno.Themes/blob/530204ba38264e66478eebdd2abc9ce05df3b0b9/crosstargeting_override.props.sample#L15

And then add a section at the bottom with a description of what it does, like this one: 
https://github.com/unoplatform/Uno.Themes/blob/530204ba38264e66478eebdd2abc9ce05df3b0b9/crosstargeting_override.props.sample#L32-L37